package.json

```diff
@@ -50,7 +50,7 @@
     "rimraf": "^2.5.2"
   },
   "ember-addon": {
-    "configPath": "tests/dummy/config",
```

Comment:
Hmm, I wouldn’t think we would need to change this (but the config/deprecations.js would need to be moved to tests/dummy/config)...